libraries/rush-lib/webpack.config.js

```diff
@@ -121,10 +121,6 @@ module.exports = () => {
       [PathConstants.installRunRushPnpmScriptFilename]: {
         import: `${__dirname}/lib-esnext/scripts/install-run-rush-pnpm.js`,
         ...SCRIPT_ENTRY_OPTIONS
-      },
-      [PathConstants.createLinksScriptFilename]: {
```

Comment:
I'm confused. This project isn't getting bundled, and that file is intended to be a utility file that simply gets dropped into the output of the deploy. It doesn't export any modules since it's just a script file that is intended to be run directly.

Reply:
All of the utility scripts in rush-lib were bundled before to deal with any current or future imports.

I can look at and make this change when I get back, though.